### DISCUSSION TITLE
JellyPlayer Alpha 1

### DISCUSSION THREAD

**Discussion by prayag17** [prayag17/Blink#114] [Announcements]
JellyPlayer's alpha 1 has been released, executables should appear in some time.

There are a ton of bugs and a lot is subject to change. You can track Alpha 2 [here](https://github.com/users/prayag17/projects/4/views/1) 

Features can be requested by going [here](https://github.com/prayag17/JellyPlayer/issues)

The next release will focus on improving the whole client performance and video playback, adding SkipIntro, music playback as well as improving the UI

**Comment by Sparklingx** [comment]
this is so cool. love the client. Full support 🔥🔥💖

**Reply by jackloves111** [reply]
3 weeks ago，i am Look forward to update

**Reply by prayag17** [reply]
Yeah it will take me some time to release Alpha 2 as i have a lot of exams this year

**Reply by Sparklingx** [reply]
when Jelly player(based of jellyfin vue) or jellyfin vue web client is fully ready I will permanently shift from emby/plex.

**Reply by prayag17** [reply]
JellyPlayer is not based on VUE client. It is custom made in react

**Reply by Sparklingx** [reply]
Ah  I see. both looks about same so I thought it may be based on vue. but whatever jelly player is so cool.

**Reply by prayag17** [reply]
Yeah that's cause I took inspiration from it and also cause of material ui
